exaudfclient/base/exaudflib/exaudflib.cc

```diff
@@ -21,57 +19,30 @@
 #include <limits>
 #include "exaudflib/zmqcontainer.pb.h"
 #include "script_data_transfer_objects_wrapper.h"
-#include <unistd.h>
 
-#include <mutex>
 
-#ifdef PROTEGRITY_PLUGIN_CLIENT
-#include <protegrityclient.h>
-#endif
+#include "exaudflib/impl/check.h"
+#include "exaudflib/impl/socket_info.h"
+#include "exaudflib/impl/socket_low_level.h"
+#include "exaudflib/impl/msg_conversion.h"
+#include "exaudflib/impl/global.h"
+#include "exaudflib/socket_high_level.h"
 
-using namespace SWIGVMContainers;
-using namespace std;
-using namespace google::protobuf;
+#include "exaudflib/impl/swig/swig_meta_data.h"
+#include "exaudflib/impl/swig/swig_result_handler.h"
+#include "exaudflib/impl/swig/swig_table_iterator.h"
 
-#ifndef PROTEGRITY_PLUGIN_CLIENT
-__thread SWIGVM_params_t* SWIGVMContainers::SWIGVM_params; // this is not used in the file, but defined to satisfy the "extern" requirement from exaudflib.h
-#endif
+#include "exaudflib/vm/swig_vm.h"
 
 
-static SWIGVM_params_t * SWIGVM_params_ref = nullptr;
 
+#ifdef PROTEGRITY_PLUGIN_CLIENT
```

Comment:
do we need this, because we already include it in exaudfclient.cc and use it there for creation

Reply:
From what I have seen in protegrityclient.h we don't need it. Removed!